nasa/earthdata-search
# EDSC-4032: remove download and learn more

# Overview

### What is the feature?

Changed the language for downloading EDD. 

### What is the Solution?

Fixed up the languages, removed the downloads that would show up upon detection of the OS and that EDD is not downloaded, and removed the tests for those features.

### What areas of the application does this impact?

List impacted areas.

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. go to http://localhost:8080/search
2. select any collection and add it to your project
3. click Download Data on the project page
4. Click on the Download Files button
5. Observe the following modal
<img width="1706" alt="image" src="https://github.com/nasa/earthdata-search/assets/8591291/9f3167fb-4012-4e01-8a35-6a3680ff1f3c">


### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.
<img width="1706" alt="image" src="https://github.com/nasa/earthdata-search/assets/8591291/62f41110-4ca9-4ec3-b3bc-fab2520c6b2e">
# Checklist
